Please run `conda env list` to see a list of all available environments. Use
`source activate <env>` to activate the environment '<env>'. 
Lmod has detected the following error: The following module(s) are unknown:
"cuda/cuda-11.3"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "cuda/cuda-11.3"

Also make sure that all modulefiles written in TCL start with the string
#%Module




Currently Loaded Modules:
  1) anaconda/anaconda3

 


CommandNotFoundError: Your shell has not been properly configured to use 'conda deactivate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


/home/pnwaorgu/my-envs/autd/lib/python3.7/site-packages/gym/logger.py:30: UserWarning: [33mWARN: Box bound precision lowered by casting to float32[0m
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
Traceback (most recent call last):
  File "./discrete/run/teacher/dungeon_quest_teacher_rew_per_step_7_productMDP.py", line 38, in <module>
    run_training(config)
  File "/lustre/fs1/home/pnwaorgu/Local_Neuro_Symbolic/discrete/lib/main.py", line 45, in run_training
    train_agent(config, agent, automaton, ap_extractor, rollout_buffer, logger, start_iter, run_name=run_name)
  File "/lustre/fs1/home/pnwaorgu/Local_Neuro_Symbolic/discrete/lib/training.py", line 558, in train_agent
    target_agent = agent.create_target_agent(tau=config.tau)
TypeError: create_target_agent() got an unexpected keyword argument 'tau'

real	0m16.259s
user	0m13.162s
sys	0m4.638s
mv: cannot stat â€˜./test_output/test_output*â€™: No such file or directory
